deepchem/feat/material_featurizers/lcnn_featurizer.py

```diff
@@ -0,0 +1,745 @@
+import numpy as np
+from deepchem.feat import MaterialStructureFeaturizer
+from collections import defaultdict
+from typing import List, Dict, Tuple, DefaultDict, Any
+
+
+class LCNNFeaturizer(MaterialStructureFeaturizer):
+  """
+  Calculates the 2-D Surface graph features in 6 diffrent permutaions-
+
+  Based on the implementation of Lattice Graph Convolution Neural
+  Network (LCNN). This method produces the Atom wise features ( One Hot Encoding)
```

Comment:
Can you add a bit of explanation here about what advantages LCNN provides over other featurization methods and in what situations you would want to use it?